tools/wasi/wasi.cpp

```diff
@@ -150,15 +145,15 @@ bool run(int argc, const char** argv)
         return false;
     }
     const auto wasm_binary =
-        fizzy::bytes(std::istreambuf_iterator<char>{wasm_file}, std::istreambuf_iterator<char>{});
+        bytes(std::istreambuf_iterator<char>{wasm_file}, std::istreambuf_iterator<char>{});
 
-    auto module = fizzy::parse(wasm_binary);
-    auto imports = fizzy::resolve_imported_functions(*module, wasi_functions);
-    auto instance = fizzy::instantiate(
-        std::move(module), std::move(imports), {}, {}, {}, fizzy::MemoryPagesValidationLimit);
+    auto module = parse(wasm_binary);
```

Comment:
I actually liked having the explicit fizzy prefix on things here, but I'm fine either way.

Reply:
It does not make sense when we are in `fizzy::wasi`.